src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Program.cs

```diff
@@ -48,7 +56,9 @@ private void DoTheRun(int iterations)
                     {
                         try
                         {
-                            RunAllStressTests();
+                            //RunAllHttpStressTests();    //  for TestHelpers.SetHostAndProtocol(useHttp: true
+                            RunAllNetTcpStressTests();  //  for TestHelpers.SetHostAndProtocol(useHttp: false
```

Comment:
Why are the Http tests commented out?  If this is frequently necessary we should find a mechanism (#if or an xunit trait or a TestProperty so we can enable/disable tests as needed.  Not required to merge this PR.